docs/netbsd/README.md

```diff
@@ -49,10 +67,15 @@ configure ssh.
 	PermitRootLogin without-password
 	```
 
-4. Copy your public key to `/root/.ssh/authorized_keys` on the guest and `reboot` the
+4. Now you should be able to ssh into the netbsd VM.
+	```sh
+	$ ssh -p 10022 root@127.0.0.1
+	```
+
+5. Copy your public key to `/root/.ssh/authorized_keys` on the guest and `reboot` the
```

Comment:
These instructions are not quite working, they miss some steps. Let's make them fully copy-pastable without requiring everybody to figure out non-trivial missing steps.
To begin with we need to generate the right key name e.g.:
```
ssh-keygen -f netbsd -t rsa -N ""
```

Then also how do we copy the key before we setup scp? For me it was "sorry, you lost me here".
@krytarowski showed the trick with the second disk:
```
$ cat build.sh

#!/bin/sh

set -x

num=$1
shift

tar cvvf netbsd.$num.tar $@
$ cat boot.sh

#!/bin/sh

qemu="qemu-system-x86_64 \
        -smp 1 \
        -curses \
        -m 512 \
        -hda ./nbsd.img \
        -net nic \
        -net user \
        -drive file=./nbsd.img,if=none,id=id0 \
        -device ide-drive,drive=id0,serial=id0,bootindex=0 \
        -gdb tcp::1234"

if [ "$1" = "" ]; then
        $qemu
else
        $qemu \
        -drive
file=./netbsd.$1.tar,if=none,id=id1,cache=writeback,format=raw \
        -device virtio-blk,drive=id1,serial=id1,bootindex=1
fi


And later:

$ ./build.sh 1 netbsd
$ ./boot.sh 1

and inside the machine:

$ tar xvf /dev/rld0
```

Let's describe this here.

Reply:
Also I did not find any editor that I can use, so I could not edit the /etc files. I figured out I can do:
```
export PKG_PATH=http://ftp.netbsd.org/pub/pkgsrc/packages/NetBSD/amd64/8.0/All
pkg_add nano
```
That's also useful to note.

Or otherwise we need copy-pastable commands that do the necessary edits to the files using the installed commands (echo, sed, or what else is there).